### ISSUE TITLE
Create a runbook for Mobile Vet Center changing Vet Centers

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13623]
re: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13469#issuecomment-1536480004

Mobile vet centers can be sold between Vet Centers. When this takes place,we think they show up as dupe / new facilities with new Facility API IDs. Needs a runbook similar to the Section Change runbook for VAMCs to make clear the steps that should be performed: [runbook-vamc-facility-duplicate-or-section-change.md](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/.github/ISSUE_TEMPLATE/runbook-vamc-facility-duplicate-or-section-change.md).

If the steps are basically the same, we could retrofit the VAMC dupe / section change to be for either VAMC or MVCs. 

## ACs

- [ ] Gather Helpdesk input re: steps they're aware of and whether the process mimics VAMCs enough to modify the duplicate/section change runbook, or create a new MVC-specific runbook
- [ ] Create runbook as github issuetype
- [ ] Update runbook documentation (currently in [Google doc](https://docs.google.com/spreadsheets/d/1fkGH1PGXXPHObsDc_kVnea-JQVnS0HGVKP16aBTjdpw/edit#gid=0), will move to Github link)

**Comment by jilladams** [comment]
@stefaniefgray We discussed that this has happened a couple of times before (MVCs getting sold and therefore their data changing), but we _didn't_ identify an example we could look at for reference of what all may need to change.
 
But: after we talked last week, I updated the Vet Center name change runbook ([runbook-vet-center-name-change.md](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/.github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md#drupal-admin-steps-cms-engineer-or-help-desk)) to add a step specifically re: Mobile Vet Centers changing hands. Does this seem like it will cover the bases for this sort of change? Or do additional things need to happen that would justify making this a freestanding runbook?
